src/reducers/sources.js

```diff
@@ -440,6 +440,14 @@ export function getSourcesUrlsInSources(state: OuterState, url: string) {
   return [...new Set(Object.keys(urls).filter(Boolean))];
 }
 
+export function getHasSiblingOfSameName(state: OuterState, source: ?Source) {
```

Comment:
I think this function is in the `SourcesTree` file, so I think we should move it somewhere in `utils/source.js`

Reply:
Ooops, missed the change about it :f ;